provisioner/powershell/provisioner.go

```diff
@@ -89,6 +89,9 @@ type Config struct {
 	// A duration of how long to pause after the provisioner
 	PauseAfter time.Duration `mapstructure:"pause_after"`
 
+	// Run pwsh.exe instead of powershell.exe - latest version of powershell.
+	Pwsh bool `mapstructure:"pwsh"`
```

Comment:
As noted by @JenGoldstrich in a out-of-band discussion, `UsePwsh` would be a clearer name for the attribute, could you amend this?